src/plugins/azgenerator/TemplateAzureCliParams.ts

```diff
@@ -60,7 +60,7 @@ export function GenerateAzureCliParams(model: CodeModelAz): string[] {
     output.push(")");
 
     if (hasActions) {
-        output.push("from azext_" + model.Extension_NameUnderscored + ".action import (")
+        output.push("from azext_" + model.Extension_NameUnderscored + ".actions import (")
```

Comment:
fixed in different way in another pr. correct filename is "action.py"

Reply:
but  in generator.py it still using actions.py

Reply:
"action.py" is used now. Zim, please help to pull and merge if needed. thanks